cdap-ui/app/hydrator/templates/partial/node-config-modal/node-config/plugin-edit-input-schema.html

```diff
@@ -21,9 +21,8 @@ <h4 ng-if="!HydratorPlusPlusNodeConfigCtrl.showNewSchemaEditor">Input Schema</h4
       <plugin-schema-editor
         disabled="true"
         schemas="HydratorPlusPlusNodeConfigCtrl.state.node.inputSchema"
-        on-schema-change="HydratorPlusPlusNodeConfigCtrl.onSchemaChange"
-        actions-dropdown-map="HydratorPlusPlusNodeConfigCtrl.getActionsDropdownMap(true)"
-        schema-title="'Input Schema'">
+        schema-title="'Input Schema'"
+        errors="HydratorPlusPlusNodeConfigCtrl.inputSchemaErrors">
```

Comment:
should onSchemaChange and actionsDropdownMap be marked as optional props in the plugin schema editor?